plugins/smart_proxy_remote_execution_ssh/control

```diff
@@ -10,6 +10,6 @@ XS-Ruby-Versions: all
 Package: ruby-smart-proxy-remote-execution-ssh
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
-Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.9.0~rc3), ruby-smart-proxy-dynflow (>= 0.0.4), ruby-smart-proxy-dynflow (<< 0.1.0), ruby-net-scp, ruby-net-ssh
+Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter, foreman-proxy (>= 1.11.0~rc3), ruby-smart-proxy-remote-execution-ssh-core (= 0.1.3), ruby-smart-proxy-dynflow (>= 0.1.0), ruby-smart-proxy-dynflow (<< 0.2.0), ruby-net-scp, ruby-net-ssh
```

Comment:
instead of (= 0.1.3), something like `(>= 0.1.3)` and `(<0.1.4)` is better, I'm unsure if 0.1.3-1 would be suffcient for =0.1.3